apps/dapp-console/app/faucet/components/SuccessDialog.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react'
+import { Text } from '@eth-optimism/ui-components/src/components/ui/text/text'
+import {
+  useRive,
+  UseRiveParameters,
+  Layout,
+  Fit,
+  Alignment,
+} from '@rive-app/react-canvas'
+import { Button } from '@eth-optimism/ui-components/src/components/ui/button/button'
+
+const SuccessDialog: React.FC = () => {
+  const { RiveComponent } = useRive({
+    src: '/sunny-animation.riv',
```

Comment:
Oooo Rive implementation looks pretty straightforward, is all the interaction logic passed via Rive?

Reply:
for this animation yes! it may get more complicated if we want different triggers to play with the animation but their documentation is pretty straight forward